src/components/modal/ttt/modal.jsx

```diff
@@ -0,0 +1,121 @@
+var FormattedMessage = require('react-intl').FormattedMessage;
+var MediaQuery = require('react-responsive');
+var omit = require('lodash.omit');
+var React = require('react');
+
+var FlexRow = require('../../flex-row/flex-row.jsx');
+var frameless = require('../../../lib/frameless');
+var Modal = require('../base/modal.jsx');
+var TitleBanner = require('../../title-banner/title-banner.jsx');
+
+require('../../forms/button.scss');
+require('./modal.scss');
+
+var TTTModal = React.createClass({
+    propTypes: {
+        title: React.PropTypes.string.isRequired,
+        description: React.PropTypes.string.isRequired,
+        tutorialLoc: React.PropTypes.string.isRequired,
+        activityLoc: React.PropTypes.string.isRequired,
+        guideLoc: React.PropTypes.string.isRequired,
+        thumbUrl: React.PropTypes.string.isRequired,
+        bannerUrl: React.PropTypes.string.isRequired,
```

Comment:
This is the trailing comma Travis is flagging about, (and pardon my ignorance) I believe that it's a typo?